src/mxarray.jl

```diff
@@ -658,7 +658,7 @@ function jstring(mx::MxArray)
     tmp = Array(UInt8, len)
     ccall(_mx_get_string, Cint, (Ptr{Void}, Ptr{UInt8}, mwSize), 
         mx.ptr, tmp, len)
-    bytestring(pointer(tmp))
+    unsafe_string(pointer(tmp))
```

Comment:
If I use `String(tmp)` instead I get an extra characters at the end of the string  e.g. `\0\0` .